content/applications/sales/sales/ebay_connector/deletion_notifications.rst

```diff
@@ -0,0 +1,71 @@
+===============================================
+eBay Marketplace account deletion notifications
+===============================================
+
+Since September 2021, **eBay requires supporting customer account deletion/closure notifications**.
+As such, when eBay receives an account request for deletion, all eBay partners must confirm the
+reception of the request and take further action if necessary.
+
+Odoo has a notification endpoint to receive those notifications and handle the first set of actions
```

Comment:
```suggestion
Odoo has a notification endpoint to receive those notifications, confirm the reception of the request, and handle the first set of actions
```
Otherwise, the previous paragraph leads to believe the user has to confirm something.